### ISSUE TITLE
Website Spell Check: manual

### ISSUE THREAD

**Issue by Biuwa** [hackforla/tdm-calculator#1146]
### Overview
We need to manually spell check the website so that we have a list of misspellings for development to use to test automated spell checking linters.

### Action Items
- [ ] Create a spreadsheet with the following tabs and schema
  - [ ] Tab name: Categories
    - [ ] Schema: Name of page, page nickname, page url
  - [ ] Tab name: Spell Check
    - [ ] Schema: error #, page it appears on (drop down), misspelling, correct spelling
- [ ] Find a chrome plugin
- [ ] check each page using a chrome plugin adding each error to its own line on the Spell Check tab
- [ ] Release dependency on #413 

### Resources/Instructions
- [VS Code Plugins for spelling](https://marketplace.visualstudio.com/search?term=spell&target=VSCode&category=All%20categories&sortBy=Relevance)
- [Web Based Chrome Extension checker](https://chrome.google.com/webstore/detail/spell-check/ofbiibdnnhllfldbglbhmiojndiepejl?u_producttype=chrome-extension-759435&u_country=us&funnel=esign&gclid=CjwKCAjwzt6LBhBeEiwAbPGOgWBYIwbWhghZ-uWgoMQS16y7t8pHC7kKXM0RQjeg27To0YlYVBisdRoCueMQAvD_BwE)
- [Web based external checker](https://www.icwebdesign.co.uk/how-to-spell-check-a-whole-website) - more sensitive than the chrome extension however will only check pages that are linked to main page (not about page, etc.)
- https://www.w3.org/2002/01/spellchecker (check each page) only works on pages like about not https://tdmdev.azurewebsites.net/calculation/2

#### Data validation code
```
'Categories'!$B$2:$B
```
#### History
After noticing typos and misspellings at the stakeholders' meeting on 5/4/22, Bonnie suggested the entire website be spell checked.

**Comment by SMR9220** [comment]
Will spell check be done manually or through a program?

**Comment by Biuwa** [comment]
#975

**Comment by Biuwa** [comment]
https://docs.google.com/spreadsheets/d/1b0hF107x_i_hz0AKsnOPAGoqFvhqVxa03PMmneCOcqw/edit?usp=sharing

**Comment by fnaneberhane** [comment]
I completed a manual check and spelling check using the chrome plugin. The only error I found was on the [checklist linked ](https://tdm.ladot.lacity.org/faqs)at the bottom right corner of the website. Thought it would easier to just display the error below instead of creating a spreadsheet for it, but happy to create one for logging purposes.

Error: Residentian (should say residential)

<img width="962" alt="Screenshot 2023-08-30 at 11 35 35 PM" src="https://github.com/hackforla/tdm-calculator/assets/140571786/cee8412d-a9a5-4231-abc0-a5d91d319567">
